src/view/list-filter.js

```diff
@@ -0,0 +1,25 @@
+export const createListFilterTemplate = () => {
+  return `
+  <div class="trip-controls__filters">
+  <h2 class="visually-hidden">Filter events</h2>
```

Comment:
Поправь табуляцию.
Вообще для наглядности код с возвращением строк можно оформлять след. образом:
```suggestion
return (
    `<div class="trip-controls__filters">
        ...
    </div>`
);
```